### ISSUE TITLE
Operator precedence issue with constraints to constants at priorities

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/Anchorage#30]
```
AspectImageCell.swift:25:44: error: binary operator '~' cannot be applied to operands of type 'Int' and 'Float'
                imageView.widthAnchor == 0 ~ UILayoutPriorityRequired - 1
                                         ~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

You can work around this with parentheses and casting, but it would be nice if we could fix it with operator precedence. Here's a workaround:

```swift
imageView.widthAnchor == 0 ~ UILayoutPriority(UILayoutPriorityRequired - 1)
```

**Comment by jvisenti** [comment]
@ZevEisenberg This is on the latest release (3.1.0)?

**Comment by ZevEisenberg** [comment]
@jvisenti after a brief moment of mild panic, I have determined that I am indeed on the latest release. Guess it's just a case that we missed.

**Comment by jvisenti** [comment]
This will be addressed in 4.0. See #37

**Comment by jvisenti** [comment]
Fixed in 4.0
